data/national/ceds/data_raw/README.txt

```diff
@@ -1,3 +1,47 @@
+----------------------
+v_2025_03_18 (CMIP7 final final)
```

Comment:
I'll believe that when I see it. I'd be tempted to just do the below as I think it is clearer while we're still unsure how many revisions there will be. Alternately, we just keep adding an extra final as the versions roll through and see how far we have to go 🙃 

```suggestion
v_2025_03_18
```

Reply:
Haha I originally actually took this naming from the ScenarioMIP Google Drive, where 11 March was called 'final', and 18 March was called 'final final'.
Anyway - like this is of course clear enough for our purposes now, especially because it now has the Zenodo entry.